sessionctx/variable/sysvar.go

```diff
@@ -681,6 +681,16 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal, Name: TiDBStatsCacheMemQuota, Value: strconv.Itoa(DefTiDBStatsCacheMemQuota), skipInit: true,
+		MinValue: 0, MaxValue: math.MaxInt64, Type: TypeInt,
```

Comment:
You don't need to specify skipInit. That has a special meaning for session scoped sysvars.

Reply:
Also: is it possible to set the MaxValue to something like 1G? (or whatever makes sense as a ceiling that exceeds what is useful). We have had some problems in the past with misconfiguration, and try to keep max values to safe ranges.

Reply:
However, I have no idea for a proper ceiling value for this variable for now.